config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -140,7 +140,9 @@ job_cluster() {
   local repo=$1
 
   case "$repo" in
-   external-snapshotter|external-resizer|lib-volume-populator|livenessprobe|node-driver-registrar|volume-data-source-validator)
+   external-snapshotter|external-resizer|lib-volume-populator|livenessprobe|node-driver-registrar|volume-data-source-validator| \
+   external-health-monitor|external-provisioner|external-attacher|csi-test|csi-release-tools|csi-proxy|csi-lib-utils| \
+   csi-driver-windows-poc|csi-driver-smb|csi-driver-nvmf|csi-driver-nfs|csi-driver-iscsi|csi-driver-host-path)
```

Comment:
Given that this now list all (?) or at least a majority of the repos, would it make sense to invert the logic and explicitly list those which *don't* use `eks-prow-build-cluster`?

Reply:
Yeah, that's a much better idea. I'll update.